pkg/ts/core/Arc.ts

```diff
@@ -63,25 +64,25 @@ export class Arc extends EventEmitter {
   addStore(storeId, store) {
     if (store && !this.stores[storeId]) {
       this.stores[storeId] = store;
-      store.listen('change', () => this.storeChanged(storeId, store));
+      store.listen('change', () => this.storeChanged(storeId, store), this.id);
     }
   }
   removeStore(storeId) {
-    if (this.stores[storeId]) {
-      // TODO(sjmiles): must `unlisten` to match `listen` above
-      //this.stores[storeId].onChange = null;
+    const store = this.stores[storeId];
+    if (store) {
+      store.unlisten('change', this.id);
     }
     delete this.stores[storeId];
   }
   // TODO(sjmiles): 2nd param is used in overrides, make explicit
   protected storeChanged(storeId, store) {
     this.log(`storeChanged: "${storeId}"`);
-    values(this.hosts).forEach((host: Host) => {
+    const isBoundBackwardCompatible = bindings => bindings && entries(bindings).some(([n, v]) => (v || n) === storeId);
+    const isBound = inputs => inputs && inputs.some(input => values(input)[0] == storeId || keys(input)[0] == storeId);
+    values(this.hosts).forEach(host => {
       const bindings = host.meta?.bindings;
       const inputs = host.meta?.inputs;
-      const isBoundBackwardCompatible = (bindings) => bindings && entries(bindings).some(([n, v]) => (v || n) === storeId);
-      const isBound = (inputs) => inputs && inputs.some(input => values(input)[0] == storeId || keys(input)[0] == storeId);
-      if (isBoundBackwardCompatible(bindings) || isBound(inputs)) {
+      if (bindings === '*' || isBoundBackwardCompatible(bindings) || isBound(inputs)) {
```

Comment:
somewhat related - i was hoping we could deprecated `bindings` sometimes soon, leaving only `inputs` and `outputs`. could we use `inputs` here instead of `bindings`?

Reply:
This syntax only allows for '*', I'm curious if we might want to mix '*' + $inputs, so I'm leaving it as 'bindings'

If you want to remove support for `bindings` meaning anything else, I think that's ok!